crates/wasmtime/src/trap.rs

```diff
@@ -183,7 +183,54 @@ pub struct WasmBacktrace {
 }
 
 impl WasmBacktrace {
-    fn new(
+    /// Captures a trace of the WebAssembly frames on the stack for the
+    /// provided store.
+    ///
+    /// This will return a [`WasmBacktrace`] which holds captured
+    /// [`FrameInfo`]s for each frame of WebAssembly on the call stack of the
+    /// current thread. If no WebAssembly is on the stack then the returned
+    /// backtrace will have no frames in it.
```

Comment:
Maybe also mention that this will only contain Wasm frames from within the given store, not other stores' frames that might be on the stack.